projects/plone-restapi-angular/src/lib/directives/download.directive.ts

```diff
@@ -46,7 +46,7 @@ export class DownloadDirective {
   private static saveDownloaded(namedFile: NamedFile, blob: Blob) {
     const a = window.document.createElement('a');
     a.href = window.URL.createObjectURL(
-      blob, <ObjectURLOptions>{ type: namedFile['mime-type'] }
+        new Blob([blob], { type: namedFile['mime-type'] })
```

Comment:
because `ObjectURLOptions` doesn't exist anymore in new version of typescript, this is how we can get a blob with the mime-type (solution coming from @ebrehault, he might have more details on how it works internally).